src/search/containers/CurrentSearchPage.js

```diff
@@ -9,10 +9,10 @@ export function mapStateToProps(state) {
 
 export function mapDispatchToProps(dispatch) {
   return {
-    handleSearch: (query) => {
-      dispatch(searchStart(query));
-      return search({ query })
-        .then((response) => dispatch(searchLoad({ query, ...response })));
+    handleSearch: (query, page = 1) => {
```

Comment:
This is enforced by eslint. I filed https://github.com/mozilla/addons-frontend/issues/164 to fix it later.